1.7/administration/installing/cloud/aws/index.md

```diff
@@ -1,92 +1,19 @@
 ---
-post_title: AWS DC/OS Installation Guide
-nav_title: AWS
+layout: page
+post_title: Installing DC/OS on AWS
 menu_order: 0
 ---
 
-You can create a DC/OS cluster for Amazon Web Services (AWS) by using the DC/OS template on AWS CloudFormation.
 
-Depending on the DC/OS services that you install, you might have to modify the DC/OS templates to suit your needs. For more information, see [Scaling the DC/OS cluster in AWS][1].
+You can create a DC/OS cluster for Amazon Web Services (AWS) by using the DC/OS templates for AWS CloudFormation.
 
-# System requirements
+# Basic
+The basic AWS CloudFormation template creates a DC/OS cluster that is suitable for demonstrations and POCs. This is the fastest way to get started with the DC/OS templates for AWS CloudFormation.
 
-## Hardware
+- [Basic AWS Install Guide](/docs/1.7/administration/installing/cloud/aws/basic/)
 
-An Amazon EC2 <a href="https://aws.amazon.com/ec2/pricing/" target="_blank">m3.xlarge</a> instance.  Selecting smaller-sized VMs is not recommended, and selecting fewer VMs will likely cause certain resource-intensive services, such as distributed datastores, to not work properly.
+# Advanced
+The advanced AWS CloudFormation templates bring power and flexibility to creating and extending DC/OS clusters. With these templates you can choose from the complete set of DC/OS configuration options.
```

Comment:
Instead of "choose from the complete set", I'd like a list of 3 or 4 things you get to change.